views/index.pug

```diff
@@ -1,11 +1,51 @@
 extends layout
+block content
+  h1.title= title
+    p.descriptor See what your friends are watching and rate movies you watch!
+    if !locals.authenticated
+      div.bestOfBox
+        h2 1st Annual Marvelous Movies Random Movie Generator
+        a.submit(href='/movies/' + bestFilm.id) Feeling Lucky?!<br>(click for a random movie)
+        h2 See top Movies by category
+        div.bestOfBox2
+          a(href='/movies/genre/Action') Best Action
```

Comment:
Could you pass all of the genres to the `index.pug` template an iterating over them to render these links?